CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 ### Performance enhancements
 
 ### Bug fixes
+* In `skbio.stats.composition`, all instances of `np.apply_along_axis` were replaced with list comprehensions for backwards compatibility with `namedtuple`s returned by scipy statistical tests in scipy >=0.18 as well as the `namedtuple`s from `np.apply_along_axis` in numpy >=1.12.0. See ([#1472](https://github.com/biocore/scikit-bio/issues/1472)).
```

Comment:
The link to the original PR can be found in the CHANGELOG.md and the PR in numpy that introduced these issues can be found at the top.  So all of the links are already in place.  Do you think it is still necessary to add this into the CHANGELOG?